### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by toddmotto**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1 +1,33 @@
...
+- Define the core structure of an Angular application
+- Create a feature based file structure
+
+## Core Structure
+
+Angular supports modules, and our application file structure will reflect this. Each module will have directives, templates, controllers and services.
+
+Like a normal application, we'll have separate folders for our JavaScript, CSS and images. However, inside our JavaScript folder we'll have individual folders for each module in our Angular application. Angular needs at least one module to kick things off, and we're going to call our main module "app".
+
+This means that you'll end up with a directory structure like this -
```
Comment (ipc103) [inline README.md]: It's confusing to see this file structure with a bunch of empty folders. Show an actual, finished app with this structure first and include a note like "don't worry about what all of these files and folders do yet, we'll be discussing that in future lessons"

File: README.md [hunk 2]
```diff
@@ -1 +1,33 @@
...
+We need to ensure that we have a level-headed way of structuring our applications from the beginning, otherwise we could end up with files everywhere, which means it's hard to know exactly what you are editing and where.
+
+## Objectives
+
+- Define the core structure of an Angular application
+- Create a feature based file structure
+
+## Core Structure
+
+Angular supports modules, and our application file structure will reflect this. Each module will have directives, templates, controllers and services.
```
Comment (ipc103) [inline README.md]: What do you mean by modules? Define this more clearly. Also, we don't know what controllers, services, directives, or templates are yet. Either explain it, or let the reader know they don't have to worry about this yet.

File: README.md [hunk 3]
```diff
@@ -1 +1,33 @@
...
+    - services/
+    - templates/
+      - partials/
+      - views/
+  - angular.js
+- img/
+- css/
+- index.html
+
+This is a feature based file structure, as we're splitting off into individual modules instead of having all of our controllers/directives/etc. in one folder.
```
Comment (ipc103) [inline README.md]: What makes this feature based? What would the alternative be? Do we _have_ to structure our application this way, or is it best practice?

**Comment by ipc103** [comment]
This is straightforward, but it's hard to understand why to use this directory structure or what the alternative will be. Consider giving a brief explanation of each directory and what it does. For example, "Our controllers directory holds all of the code related to retrieving some data and passing it into the view." 

If it's important to see this now, give a little bit more context. "You don't need to memorize this structure right now, but we want you get a sense of what our project will be building towards."

File: README.md [hunk 4]
```diff
@@ -1 +1,65 @@
...
+- Define the core structure of an Angular application
+- Create a feature based file structure
+
+## Core Structure
+
+Angular supports modules, and our application file structure will reflect this. Each module will have directives, templates, controllers and services.
+
+Like a normal application, we'll have separate folders for our JavaScript, CSS and images. However, inside our JavaScript folder we'll have individual folders for each module in our Angular application. Angular needs at least one module to kick things off, and we're going to call our main module "app".
+
+This means that you'll end up with a directory structure like this -
```
Comment (ipc103) [inline README.md:46]: Really like seeing the files inside of the folders. Because this ends up looking very similar to the non-feature-based file structure, can you show an example of an app that has two modules?

File: README.md [hunk 5]
```diff
@@ -1 +1,65 @@
...
+    - app.js
+  - angular.js
+- img/
+- css/
+- index.html
+
+This is a feature based file structure, as we're splitting off into individual modules instead of having all of our controllers/directives/etc. in one folder. Don't be scared - we will be touching on controllers/directives/modules/etc very soon!
+
+There are plenty of other ways to structure your application. One other popular way is to just have folders for controllers, directives etc without having folders for each module, as follows - 
+
```
Comment (ipc103) [inline README.md]: Love seeing the other file structure - again, show us what this would look like with more than one "module" so we can make a better comparison

File: README.md [hunk 6]
```diff
@@ -1 +1,65 @@
...
+        - TodoForm.html
+      - views/
+        - Todo.html
+    - app.js
+  - angular.js
+- img/
+- css/
+- index.html
+
+This is a feature based file structure, as we're splitting off into individual modules instead of having all of our controllers/directives/etc. in one folder. Don't be scared - we will be touching on controllers/directives/modules/etc very soon!
```
Comment (ipc103) [inline README.md]: Love the context here that we're going to be covering more soon

File: README.md (hunk 2 above)
Comment (ipc103) [inline README.md]: Try to clarify this a little bit more - what does a module represent? When learning Rails, students have a concept of a resource  ie, a blog would have posts, users, and comments. Is this what the modules have?

**Comment by ipc103** [comment]
Love seeing the files inside of the folders - much easier to get a picture of what the application will look like. Also, really nice context about what will be covered later on. For this, start of by showing the non feature based file structure first. Then, show the reader what happens when you add a second module. After than, show them the feature based file structure. This will let them connect why feature-based is better. 

Simply put, show the "wrong way" first. Then, show what happens when you do things the wrong way. After that, show the right way.

File: README.md [hunk 7]
```diff
@@ -1 +1,65 @@
...
+
+There are plenty of other ways to structure your application. One other popular way is to just have folders for controllers, directives etc without having folders for each module, as follows -
```
Comment (ipc103) [inline README.md]: Show this way first - then, you can help the reader connect why feature-based is better

File: README.md [hunk 8]
```diff
@@ -1 +1,80 @@
...
+
+## Objectives
+
+- Define the core structure of an Angular application
+- Create a feature based file structure
+
+## Core Structure
+
+There are plenty of other ways to structure your application. One popular way is to just have folders for controllers, directives etc without having folders for each module, as follows - 
+
```
Comment (ipc103) [inline README.md:15]: This is great - ToDo and Login are great examples because they're super easy to visualize

File: README.md [hunk 9]
```diff
@@ -1 +1,80 @@
...
+  - angular.js
+- img/
+- css/
+- index.html
+
+This works, but as your application gets bigger, the folders will end up with loads of files in them, without us knowing what part of the application they are responsible for.
+
+This is why we separate our app into "modules". We can then immediately know what file is used in what part of the application just by looking at what folder it is in. Each module will have directives, templates, controllers and services.
+
+Like a normal application, we'll have separate folders for our JavaScript, CSS and images. However, inside our JavaScript folder we'll have individual folders for each module in our Angular application. Angular needs at least one module to kick things off, and we're going to call our main module "app".
```
Comment (ipc103) [inline README.md:44]: change this to mention that in this example we have todos and directives, but ordinarily we may start out with just app
